include/geolib/sensors/LaserRangeFinder.h

```diff
@@ -79,41 +84,40 @@ class LaserRangeFinder {
 
     inline double getRangeMax() const { return range_max_; }
 
-    inline int getNumBeams() const { return num_beams_; }
+    inline uint getNumBeams() const { return num_beams_; }
 
-    geo::Vector3 rangeToPoint(double range, int i) const;
+    geo::Vector3 rangeToPoint(double range, uint i) const;
 
-    const geo::Vector3 getRayDirection(int i) const;
+    const geo::Vector3 getRayDirection(uint i) const;
 
     bool rangesToPoints(const std::vector<double>& ranges, std::vector<geo::Vector3>& points) const;
 
-    int getAngleUpperIndex(double angle) const;
+    uint getAngleUpperIndex(double angle) const;
```

Comment:
Could you write a docstring for what this function is supposed to return. If I understand correctly it is the index of the first ray whose angle is greater than that of the the point specified.